### PR TITLE
APPSRE-3943 add grafana dashboard and bump to python3.9

### PR DESCRIPTION
**Pull request by fishi0x01**
- migrate grafana dashboard from `app-interface` to this upstream
- fix dependency issue with `openapi-schema-validator 0.3.0a`

```
error: jsonschema 3.2.0 is installed but jsonschema<5.0.0,>=4.0.0 is required by {'openapi-schema-validator'}
```

- remove python3.6 support (reached end of life as of December 2021)

```
importlib-metadata requires Python '>=3.7' but the running Python is 3.6.8
```

- move CI into container to decouple from jenkins agent environment. If multiple dockerfiles are not desired, then I could also move it into a single multistage build file

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM        registry.access.redhat.com/ubi8/python-36
+FROM        registry.access.redhat.com/ubi8/python-38
```
Comment (patjlm) [inline Dockerfile:1]: would python-39 be an option ?

Comment (fishi0x01) [reply Dockerfile]: It seems to work smoothly with `3.9` too -> bumped the version to `3.9` now.
Set the minimum required version to `3.8`, as this is default on many platforms now.

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
       author='Red Hat Application SRE Team',
       author_email="sd-app-sre@redhat.com",
       description='',
-      python_requires='>=3.7',
+      python_requires='>=3.8',
```
Comment (patjlm) [inline setup.py:10]: 3.9 here as well ?

Comment (fishi0x01) [reply setup.py]: You are right. Actually it is better to keep that in sync with base images :+1:

Comment (patjlm) [reply setup.py]: Thanks.. i hope better python experts will not contradict now ;-)

**Comment by patjlm** [comment]
LGTM, but a review from someone more knowledgeable on dashdotdb and this python upgrade would be nice.

**Comment by rporres** [comment]
LGTM. In the absence of tests, I guess you will need to manually check that all examples work...

**Review by rporres — APPROVED** [review]

**Comment by fishi0x01** [comment]
Examples in `README` are running smoothly - so I assume this is working as expected. Will create a ticket in backlog to create tests for this project.

**Comment by rporres** [comment]
in the `examples` directory you will find more input docs that it would be great to check too. Give me a shout if you have doubts on how to use them.

**Pull request merged**